### PR TITLE
Queries that have their contexts cancelled are logged as Info, not Error...

### PR DESCRIPTION
**Pull request by aaijazi**
..., in VTGate.

Can't do better testing than this, because bson RPC's StreamGo doesn't actually use a context (and it probably doesn't make sense to add functionality to bsonrpc at this point).

@guoliang100

### CODE REVIEW

File: go/vt/tabletserver/tabletconn/tablet_conn.go [hunk 1]
```diff
@@ -25,6 +25,7 @@ const (
 
 const (
 	CONN_CLOSED = OperationalError("vttablet: Connection Closed")
+	CANCELLED   = OperationalError("vttablet: Context cancelled")
```
Comment (yaoshengzhe) [inline go/vt/tabletserver/tabletconn/tablet_conn.go]: Does golint suggest to use camel case ?

**Comment by yaoshengzhe** [comment]
LGTM. One minor comment.

**Comment by guoliang100** [comment]
I cannot find where it generates the error "Context cancelled"...

**Comment by aaijazi** [comment]
Fixed both concerns, PTAL

**Comment by aaijazi** [comment]
@guoliang100 ping

**Comment by yaoshengzhe** [comment]
LGTM

**Comment by guoliang100** [comment]
LGTM

**Pull request merged**